### ISSUE TITLE
latest version of project freezes at runtime for me when approaching any planet

### ISSUE THREAD

**Issue by spiro-angelakis** [Zylann/solar_system_demo#5]

**Comment by Zylann** [comment]
Anything showing up in the console? Errors in Godot's debugger?

**Comment by spiro-angelakis** [comment]
nevermind sorry, i believe the issue was from me using that other non-stable build of godot voxel tools i had

just grabbed the new 3.4 version and its working now

**Comment by Malkverbena** [comment]
I'm using version 3.4 of Godot and I'm also experiencing this bug.
I just downloaded and compiled the branch 3.4 few moments ago, so it's up to date.
SO: Ununtu

**Comment by Zylann** [comment]
Are you sure you are running the right version? I cannot reproduce this with the `godot3.4` branch of the module.

**Comment by Malkverbena** [comment]
Ahhhhh. Got it. I was using the master branch. with tne 3.x branch it's work.
Merci!

**Comment by Malkverbena** [comment]
Another unrelated subject.
Do you have plans to support multiplayer for this module?

**Comment by Zylann** [comment]
Yes, but at the rate things go, I will work on it only in Godot 4.
